go.mod

```diff
@@ -21,7 +21,7 @@ module github.com/apache/incubator-yunikorn-k8shim
 go 1.12
 
 require (
-	github.com/GoogleCloudPlatform/spark-on-k8s-operator v0.0.0-20200817155620-c19d2b8660d8
+	github.com/GoogleCloudPlatform/spark-on-k8s-operator v0.0.0-20201215015655-2e8b733f5ad0
```

Comment:
which version of spark-k8s-operator this line point to? 
I think we need to point to the most stable released version

Reply:
It's this version: https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/releases/tag/v1beta2-1.2.2-3.0.0. When I manually entered `v1beta2-1.2.2-3.0.0`, the go module system automatically changed it to the string in the PR